scenarios/metrics/query-promql/Makefile

```diff
@@ -3,10 +3,19 @@ NS="metrics-query-promql"
 .PHONY: all
 all: install
 
+.PHONY: helm-repo
+helm-repo:
+	helm repo add deliveryhero 'https://charts.deliveryhero.io/'
+	helm repo update deliveryhero
```

Comment:
I think we don't need this as it is now added as a dependency in the chart. Just the `helm-deps` target you added should be enough.